### ISSUE TITLE
Fechas malas en eventos.csv

### ISSUE THREAD

**Issue by ignacioarteaga99** [IIC2413/Syllabus-2022-2#100]
Inicialmente al traspasar `eventos,csv` a una tabla en mi base de datos, lo hice con las fechas como formato varchar (string). Luego, me di cuenta de que tenía que utilizarlas como fechas para utilizarlo en alguna consulta. El problema está que al intentar realizar la copia de estos datos a una tabla me tira el siguiente error: 
`ERROR:  date/time field value out of range: "30/2/2019"`

Luego de investigar harto y ver cómo manejar dicho error, me di cuenta de que tiene que ver con el hecho de que efectivamente NO existe esa fecha, el 30 de febrero de 2019 no existe ya que febrero tiene 29 días y 30 únicamente en año bisiesto (... 2012, 2016, 2020, 2024...). 
¿Qué hacemos en este caso? ¿Alteramos el valor manualmente?

![image](https://user-images.githubusercontent.com/48535635/196844399-b2a3faeb-4517-4d10-91b4-32e90c9884ab.png)

**Comment by naderhadweh** [comment]
Una consulta, lo mismo ocurre con el 31/04, ya que abril tiene 30 días. En ese caso borramos los datos o los cambiamos?

**Comment by nfragav** [comment]
Sí! Disculpas. Traten de cambiar las fechas erróneas a fechas válidas. Por ejemplo, 31/04 pasarlo a 30/04.
